go.mod

```diff
@@ -15,18 +15,19 @@ require (
 	github.com/jmoiron/sqlx v1.2.0
 	github.com/lib/pq v1.8.0
 	github.com/mattermost/mattermost-operator v1.13.0
+	github.com/mattermost/rotator v0.0.0-20210225115736-5e1a4be5925b
```

Comment:
Can't wait to get this in! Let's hold off a bit and get the rotator PR merged first so we can point our module at that commit (or tag if we decide that is worth it).

Reply:
Yeap that will be blocked until the rotator gets merged and we release v0.1.0. I just created the PR to track all other changes ;)

Reply:
This should be ready to review now :)